exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcSpanExporter.java

```diff
@@ -291,7 +291,8 @@ protected Builder fromConfigMap(
       String metadataValue = getStringProperty(KEY_METADATA, configMap);
       if (metadataValue != null) {
         for (String keyValueString : Splitter.on(';').split(metadataValue)) {
-          final List<String> keyValue = Splitter.on('=').splitToList(keyValueString);
+          final List<String> keyValue = Splitter.on('=').limit(2).trimResults().omitEmptyStrings()
```

Comment:
Thanks! If you can add two more test cases, one with trimmed results and one with empty strings, that'd be great.

Reply:
added